From c07882fb6ea8a02869c84fd79e48855229ca5985 Mon Sep 17 00:00:00 2001
From: Peter Wen <wnwen@chromium.org>
Date: Wed, 18 Aug 2021 09:53:11 -0400
Subject: [PATCH] [PATCH] jinja2: add _identifier to pydeps for py3

If _identifier is deleted then build/print_python_deps.py fails to
detect _identifier.py as a pydep for jinja, resulting in the isolate
files missing third_party/jinja2/_identifier.py as a necessary file.

Bug: 1228231
---
 third_party/jinja2/lexer.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/third_party/jinja2/lexer.py b/third_party/jinja2/lexer.py
index 6fd135dd5b0a..1f790259d671 100644
--- a/third_party/jinja2/lexer.py
+++ b/third_party/jinja2/lexer.py
@@ -45,12 +45,6 @@ else:
     from jinja2 import _identifier
     name_re = re.compile(r'[\w{0}]+'.format(_identifier.pattern))
     check_ident = True
-    # remove the pattern from memory after building the regex
-    import sys
-    del sys.modules['jinja2._identifier']
-    import jinja2
-    del jinja2._identifier
-    del _identifier

 float_re = re.compile(r'(?<!\.)\d+\.\d+')
 newline_re = re.compile(r'(\r\n|\r|\n)')
--
2.33.0.rc1.237.g0d66db33f3-goog

